File: problemas/iniciante/2717/2717.cs
using System;
using System.Collections.Generic;
using System.Linq;

class URI {
    static void Main(string[] args) {
        int N = int.Parse(Console.ReadLine().Trim());
        List<int> AB = Console.ReadLine().Trim().Split(' ').Select(x => int.Parse(x)).ToList();

        if(AB[0] + AB[1] <= N){
            Console.WriteLine("Farei hoje!");
        }else{
            Console.WriteLine("Deixa para amanha!");
        }
    }
}

File: problemas/iniciante/2717/2717.go
package main

import (
    "fmt"
)

func main() {
    var n, a, b int

    fmt.Scanf("%d\n", &n)
    fmt.Scanf("%d %d\n", &a, &b)

    if a+b <= n {
        fmt.Printf("Farei hoje!\n")
    } else {
        fmt.Printf("Deixa para amanha!\n")
    }
}
